cliutil/format.go

```diff
@@ -2,24 +2,28 @@ package cliutil
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/Masterminds/semver"
 )
 
-func Format(nextVersion semver.Version, hasNextVersion bool, format string) []string {
+func Format(nextVersion semver.Version, hasNextVersion bool, format string, prefix ...string) []string {
```

Comment:
Since it doesn't make sense to have more than one prefix most, please change the `prefix` parameter to a mandatory `string`, and remove the `...` operator.

Reply:
Sure, I thought about that as an optional parameter. I have added your suggestion.